### ISSUE TITLE
Getting wrong region wheel for a game from screenscraper

### ISSUE THREAD

**Issue by joaoluizcarvalho** [muldjord/skyscraper#198]
I'm trying to scrape this SNES rom from screenscraper.fr:

name="Super Mario World 2 - Yoshi's Island (USA, Asia) (Rev 1).sfc"
size="2097152"
crc="cf98ddaa"
md5="ce1e3e33b6e39d37b43d7de599f9e785" sha1="34612a93741f156d6e497462ab7f253cb8a959a0"

Screenscraper has a japanese wheel and a world wheel:
https://www.screenscraper.fr/gameinfos.php?gameid=2163&action=onglet&zone=gameinfosmedias

I always get the japanese wheel. I already tried different approaches:
--region wor
--region us
--region ss
regionPrios="us,wor,ss,jp" (in the .ini file)

What am I doing wrong?

**Comment by muldjord** [comment]
That is odd. I'll look into it. Stay tuned.

EDIT: Found the problem. There is a faulty wheel in the screenscraper database. It is entered with `wor` region, but contains the `jp` wheel artwork. It is not listed on the website, but it is there in the returned JSON data.
The details in the JSON data even says they are the same size, which I highly doubt would be the case unless it was a duplicate copy of the entry. So I'm guessing someone was messing about with them at some point and inadvertently copied the database entry and replaced the artwork incorrectly.

EDIT2: I've reported the error to them on Discord. I don't think we can remove this faulty entry ourselves.
